analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Sonar/SymbolicExecutionAnalyzerFactoryTest.cs

```diff
@@ -88,13 +85,22 @@ public void GetEnabledAnalyzers_ReturnsEmptyList_WhenDiagnosticsAreDisabled()
             var sut = new SymbolicExecutionAnalyzerFactory();
             var diagnostics = symbolicExecutionRuleIds.ToImmutableDictionary(v => v, _ => ReportDiagnostic.Suppress);
             var context = CreateSyntaxNodeAnalysisContext(diagnostics);
-            var analyzers = sut.GetEnabledAnalyzers(context).ToList();
-            var enabledAnalyzers =
-                analyzers
-                    .SelectMany(analyzer => analyzer.SupportedDiagnostics.Select(descriptor => descriptor.Id))
-                    .ToList();
+            var analyzers = sut.GetEnabledAnalyzers(context, false, false).ToList();
+            var enabledAnalyzers = analyzers.SelectMany(analyzer => analyzer.SupportedDiagnostics.Select(descriptor => descriptor.Id)).ToList();
+
+            enabledAnalyzers.Should().BeEmpty();
+        }
+
+        [TestMethod]
+        public void GetEnabledAnalyzers_ReturnsEmptyList_OnTestProjectScannerRun()  // Old SE rules do not run on test code
```

Comment:
The comment is a bit misleading, as there are rules that are run in standalone nuget package case.